drivers/clock_control/clock_control_wch_rcc.c

```diff
@@ -90,7 +90,7 @@ static void clock_control_wch_rcc_setup_flash(void)
 	} else {
 		latency = FLASH_ACTLR_LATENCY_1;
 	}
-#elif defined(CONFIG_SOC_SERIES_CH32V00X)
+#elif defined(CONFIG_SOC_SERIES_QINGKE_V2A)
```

Comment:
The original is correct.

Reply:
To add detail: the flash latency is a SOC feature, not a core feature, and it's different on the CH32V003 to the others in the CH32V00x series.

Reply:
But this is the correct filter right? This will be CONFIG_SOC_SERIES_QINGKE_V2 soon.

Reply:
fixed